imports/importers.py

```diff
@@ -118,8 +116,7 @@ def _validate_fields(self, layer):
         :param layer: The layer to be validated
         :raise ImportValidationError: if invalid field found
         """
-        layer_fields = layer.fields[1:]  # first field reserved as deletion flag
```

Comment:
We never implemented feature deletion via shapefiles, and there's no plan to do so